### ISSUE TITLE
f51b46 should have a mapping to WCAG

### ISSUE THREAD

**Issue by ebalink** [act-rules/act-rules.github.io#1710]
The rule [f51b46](https://act-rules.github.io/rules/f51b46) should have a direct mapping to the [A-level criteria 1.2.2](https://www.w3.org/TR/WCAG21/#captions-prerecorded).
As this ACT rule covers both closed and open captions it should be regarded as a requirement for conformance.

To inapplicable test cases should be added a third example where the rule doesn't apply because the video is labeled as a media alternative for text.

**Comment by WilcoFiers** [comment]
@ebalink The answer for this issue and for https://github.com/act-rules/act-rules.github.io/issues/1709 are essentially the same.

These two rules are atomic rules, they are part of a bigger test (see Used In Rules). There are two ways to meet SC 1.2.2 Captions. Either by providing captions (which is what f51b46 tests), or if the video is itself an alternative for text-base content, which is tested in the rule `video element content is media alternative for text` (ab4d13). One of those rules is required for conformance, but not both. This is tested in the composite rule `video element auditory content has accessible alternative` (eac66b).

This particular one is admittedly a little quirky. That's because 1.2.2 is a little odd. A maybe better known example of how composite rules work is for 2.4.1 Bypass Blocks. There are a few ways to meet that; skip links, headings, landmarks, etc. Sites need to use at least one of those. There are atomic rules that check each solution individually, but the only the composite rule, which says that one of those atomic rules is passed is actually required for 2.4.1.

**Comment by ebalink** [comment]
Ok. Thank you so much for clarifying this structure of atomic and composite rules. And sorry for the extra work.

In my mind, when a media content is a media alternative for text, it would mean that the this rule would just be regarded as inapplicable. But for automation I now of course understand the logic to start the evaluation from a found element and then continue the evaluation through both of these possibilities.
